README.md

```diff
@@ -163,12 +165,26 @@ All of the pipeline scripts (including `vagrant/deploy.sh`) honour a
 used within the pipeline. This is useful for development and code review:
 
 ```
-BRANCH=$(git rev-parse --abbrev-ref HEAD) make dev
+BRANCH=$(git rev-parse --abbrev-ref HEAD) make dev pipelines
 ```
 
 ## Optionally deploy to a different AWS account
 
 See [doc/non_dev_deployments.md](doc/non_dev_deployments.md).
+To deploy to a different account, you'll need to export AWS access keys 
+and secrets for the account. eg to deploy/use the CI account:
+
+```
+export AWS_ACCESS_KEY_ID=your_ci_access_key
+export AWS_SECRET_ACCESS_KEY=your_ci_secret_key
+make ci pipelines
+```
+
+Due to the isolation between AWS accounts, when switching accounts, it's
+necessary to start with a comletely new deployment.
+
+**Note:** Different AWS accounts use different DNS names, so it'll be necessary
+to adjust some of the instructions above accordingly.
```

Comment:
I don't think you meant to add this back in. These docs have moved into the separate linked doc.